cmd/create/operatorroles/by_prefix.go

```diff
@@ -195,6 +195,10 @@ func handleOperatorRoleCreationByPrefix(r *rosa.Runtime, env string,
 			r.Reporter.Infof(fmt.Sprintf("To create a cluster with these roles, run the following command:\n"+
 				"\trosa create cluster --sts --oidc-config-id %s --operator-roles-prefix %s%s",
 				args.oidcConfigId, args.prefix, hostedCpOutputParam))
+			if args.installerRoleArn != "" && path != "" {
+				r.Reporter.Infof("When using these operator roles for a cluster, you must use installer role '%s' "+
```

Comment:
I feel like we should add a account-roles-prefix param to create cluster and set the prefix in the command above instead of adding this message :thinking:

Reply:
Do you mean adding `role-arn` flag?
I tried that approach and this is the tradeoff https://github.com/openshift/rosa/pull/1618#discussion_r1401871505